pre-docs/writing_tasks.md

```diff
@@ -602,6 +602,54 @@ For example, the following code in a metadata file describes a `provider` parame
  }
 ```
 
+### Making parameters sensitive
+
+To prevent a task parameter's value from being written to the Bolt logs in plain-text the `Sensitive` property can be associated with the parameter. This construct should be used for things like passwords, API keys, secrets, etc.
+
+There are two options for defining a task parameter as `Sensitive`:
+
+1) Specify the `"sensitive": true` property on the parameter within the JSON metadata [**PREFERRED**]:
+
+```
+{
+  "description": "Task has a sensitive property denoted by metadata",
+  "input_method": "stdin",
+  "parameters": {
+    "user": {
+      "description": "The user",
+      "type": "String[1]"
+    },
+    "password": {
+      "description": "The password",
+      "type": "String[1]",
+      "sensitive": true
+    }
+  }
+}
+```
+
+2) Wrap the parameter with `Sensitive` in the `type` property within the JSON metadata: 
```

Comment:
Is there a good reason to prefer this option over the 1st? The 1st is the only one that will work in PE, and I'd prefer to suggest only one way to do things.

Reply:
The usecase i see for this is:

A task author doesn't mark a parameter as `"sensitive": "true"` but the plan author doesn't want the parameter logged by Bolt, so wrapping it in a `Sensitive(xxx)` would prevent this from happening.

Maybe i should add that usecase to the documentation....

Reply:
Ahh sorry, didn't see this was in the `Writing Tasks` document... maybe i should remove this option and also remove the Sensitive type from the list in the `Common task data types` section?

Or maybe a note about option 1 being the only option supported by PE? 

Thoughts?

Reply:
Oh right, I didn't put together that it was listed in `Common task data types`. I'm partial to removing the option and the entry from `Common task data types`, but I may be forgetting something. @adreyer or @katelopresti do you remember why it was added? It seems like it was added when docs were reorganized.

Reply:
I think this is ok to merge, we can clean up the docs later.